### PR TITLE
Credential Manifest Guide 

### PR DESCRIPTION
**Pull request by bobbilee19**

### CODE REVIEW

File: site/docs/ssi/credential-manifests.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    }
+  }
+}
+```
+
+### List Credential Manifests
+
+Via a `GET` request to `v1/manifests`, the API will return all Manifests.
+
+Return all Manifests:
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:95]: can remove this line and end line 93 with a colon

File: site/docs/ssi/credential-manifests.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+```bash
+curl -X GET localhost:8080/v1/manifests
+```
+
+Return all Manifests associated with a specific Issuer `id`:
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:101]: To return all Manifests associated with a specific issuer, pass in Issuer's DID as the `id`:

File: site/docs/ssi/credential-manifests.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        }
+      }
+    }
+  ]
+}
+```
+
+### Get a Credential Manifest
+
+Via a `GET` request to `/v1/manifests/{id}`, pass in the `id` of the Manifest you wish to read.
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:145]: To get a specific Manifest, submit a `GET` request to `/v1/manifests/{id}` and pass in the `id` of the Manifest:

File: site/docs/ssi/credential-manifests.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+```bash
+curl -X GET localhost:8080/v1/manifests/8e612ac9-e450-45f8-ae57-f70c37d52441
+```
+
+If there is a matching Manifest `id`, you'll see the following response:
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:151]: If there is a matching Manifest, it will be returned as a JSON object:

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/credential-manifests.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+### Create a Credential Manifest
+
+<details>
+<summary>Field Definitions</summary>
+
+#### Required
+
+- `format` (object) - Formats that the Issuer can support when issuing the credential. At least one needs to be set. We currently only support `jwt_vc` for issuance. See [claim format registry](https://identity.foundation/claim-format-registry/#registry) for the definition.
+- `issuerDid` (string) - DID that identifies who the Issuer of the credential(s) will be.
+- `outputDescriptors` - Array of objects `manifest.OutputDescriptor` as defined in the Credential Manifest [Output Descripter](https://identity.foundation/credential-manifest/#output-descriptor) spec.
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:30]: can we be more specific here and not link to the spec? can use some of the language from the new intro

File: site/docs/ssi/credential-manifests.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+title: Credential Manifests
+---
+
+# Credential Manifests
+
+## What is a Credential Manifest?
+By definition, [Credential Manifests](https://identity.foundation/credential-manifest/) are a resource format that defines preconditional requirements, *Issuer* style preferences, and other facets *User Agents* (e.g digital wallets) utilize to help articulate and select the inputs necessary for processing and issuing a credential.
+
+:::note
+Credential Manifest is a draft specification being developed within the [Decentralized Identity Foundation](https://identity.foundation/) (DIF). The design work on the data model is ongoing and participants are encouraged to open issues or otherwise contribute at the [DIF-hosted Github repo](https://github.com/decentralized-identity/credential-manifest), whether as input to stable versions or as recommendations for future versions.
```
[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/credential-manifests.md:12]: the actual spec 404ing so this is the only link we have rn

[the remark this answers is not in the document]
Comment (bobbilee19) [reply site/docs/ssi/credential-manifests.md:12]: not sure what you mean by github to github, i am linking to github

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/ssi/credential-manifests.md [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+#### Optional
+- `description`	(string) - Explains what the Manifest in question is generally offering in exchange for meeting its requirements.
+- `issuerName` (string) - Human-readable name the Issuer wishes to be recognized by.
+- `name` (string) - Summarizing title for the Manifest in question.
+- `presentationDefinition` (object)	- Value of the presentation definition to use. Must be empty if `id` is present.
+- `presentationDefinitionId` (string) - The `id` of the presentation definition created with the `PresentationDefinitions` API. Must be empty if value is present.
+
+</details>
+
+Issuers create and host Credential Manifests for the credentials they offer. Assuming the DMV wants to create a Credential Manifest for their digital driver's license credential, they can use the SSI Service to do so.
```
Comment (angiejones) [inline site/docs/ssi/credential-manifests.md:48]: Can we move this above the fields definitions section? right after ### Create a Credential Manifest

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by angiejones** [comment]
@bobbilee19 hold off on this. I'm looking more into input descriptors vs output descriptors. will report back in a few

**Comment by angiejones** [comment]
> @bobbilee19 hold off on this. I'm looking more into input descriptors vs output descriptors. will report back in a few

ok I've updated the guide to focus on input descriptors as that's where the manifest specifies the required data. should be good to go now

**Comment by angiejones** [comment]
@decentralgabe can you please review this for accuracy?
https://vercel.live/open-feedback/developer-tbd-website-site-git-issue-669-set-beb610-tbd54566975.vercel.app

**Review by angiejones — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]
great guide! we should add this to the ssi service docs too

**Pull request merged**